packages/xs-extra/xapi-netdev.master/opam

```diff
@@ -12,8 +12,6 @@ install: [
   [make "install" "BINDIR=%{bin}%"]
 ]
 remove: [
-  ["oasis" "setup"]
-  [make "uninstall" "BINDIR=%{bin}%"]
```

Comment:
In general, a `make uninstall` is required because a package may install binaries and documentation that is not managed by `ocamlfind` and hence would not removed by it. If a package only installs a library, `ocamlfind` can be enough to remove it.